docs/Coding.md

```diff
@@ -0,0 +1,135 @@
+#Coding Standards
+
+The strmzi UI projects uses best practices based off the offical [React TypeScript Cheatsheet](https://react-typescript-cheatsheet.netlify.app/), with modificiations for this project. The React TypeScript Cheatsheet is maintained and used by developers through out the world, and is a place where developers can bring togehter lessons learned using TypeScript and React.
+
+## Imports
+
+Since we are using TypeScript 4.x + for this project, default imports should conform to the new standard set forth in TypeScript 2.7:
+
+```javascript
+import React from 'react';
+import ReactDOM from 'react-dom';
+```
+
+For imports that are not the default import use the following syntax:
+
+```javascript
+import { X1, X2, ... Xn } from 'package-x';
+```
+
+## View Props
+
+Since strmizi UI is made up of 2 different views (one for PatternFly, and one for Carbon) props for a component should be declared in seperate file instead of within the components source code. For example Carbon views are written in **View.carbon.tsx**, while PatternFly views are written in **View.patternfly.tsx**. In order to allow the views to share the same props, the code for the props should live in **View.props.tsx**.
+
+For props we are using **type** instead of **interfaces**. The reason to use types instead of interfaces is for consistency betweeen the views and because it's more constrained (See [Types or Interfaces](https://react-typescript-cheatsheet.netlify.app/docs/basic/getting-started/types_or_interfaces) for more clarifiation). By using types we are ensuring that both views will not deviate from the agreed upon [contract](https://dev.to/reyronald/typescript-types-or-interfaces-for-react-component-props-1408).
```

Comment:
Does this mean we're not using `prop-types`? If so, are default props defined in the component function signature or elsewhere?

Reply:
For typescript we don't need prop types as they take they are replace by type definition.  I will add documentation and example of how to set the default props.

Reply:
Added an example of how to set default props.